ArcadiaParties.Data.Abstractions/DTOs/DepartmentDTO.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ArcadiaParties.Data.Abstractions.DTOs
+{
+    public class DepartmentDTO
+    {
+        public string Name { get; set; }
```

Comment:
It will also need to contain department's id, because it will be used in application navigation.

Reply:
Id added